src/Microsoft.AspNetCore.Mvc.Core/Internal/ResourceInvoker.cs

```diff
@@ -89,6 +89,12 @@ public abstract class ResourceInvoker
                 {
                     _logger.ExecutingAction(_actionContext.ActionDescriptor);
 
+                    _logger.ExecutingAuthorizationFilters(_filters);
```

Comment:
These are currently logged as 'Trace', I feel they could be 'Debug' instead as its a common thing people typically wonder about.

Reply:
I agree Debug is better. Read here for more info on our guidance: https://github.com/aspnet/Logging/wiki/Guidelines#log-levels

> Debug 
Logs that are used for interactive investigation during development should use the Debug level. These logs should primarily contain information useful for debugging and have no long-term value. 
>
> This is the default most verbose level of logging.
>
> example: method x was called with parameter y